src/pages/Homepage/LendByCategoryHomepage.vue

```diff
@@ -374,10 +375,6 @@ export default {
 		}
 	}
 
-	&__body-link {
```

Comment:
`display: block` was making this have a really wide box when focused, switched to using a `br` instead